config/config.ini

```diff
@@ -1,6 +0,0 @@
-[auth]
-# Paste url to your server. If you're not sure, just stay with the default
-baseurl = http://98.220.5.217:50000
```

Comment:
@Akasiek 
This is the file that I accidentally uploaded ;/
If it can't be deleted, no worries! Was just wondering.
Thanks.